plugins/orchestrator-backend/src/service/ScaffolderService.ts

```diff
@@ -67,7 +68,7 @@ export class ScaffolderService {
     );
     const tmpDirs: string[] = new Array<string>();
     const stepOutput: { [outputName: string]: JsonValue } = {};
-    const workingDirectory: string = await this.getWorkingDirectory(
+    const workingDirectory: string = await getWorkingDirectory(
```

Comment:
Since `getWorkingDirectory` can throw errors, maybe handle it somehow and skip the execution?